src/WattleScript.Interpreter/Execution/VM/Processor/Processor_InstructionLoop.cs

```diff
@@ -285,6 +286,10 @@ private DynValue Processing_Loop(int instructionPtr, bool canAwait = false)
 							ref var tab = ref m_ValueStack.Peek();
 							if (top.Type != DataType.Table) throw new InternalErrorException("v-stack top NOT table");
 							if (tab.Type != DataType.Table) throw new InternalErrorException("v-stack tab NOT table");
+							if (top.Table.ModifierFlags.HasFlag(MemberModifierFlags.Static))
```

Comment:
Guard against `Get("Name").String` returning null with a `?? "(null))`. It technically shouldn't be possible, but we don't want to cause a NullReferenceException while having another exception

Reply:
fixed in 277b828